### ISSUE TITLE
Add support for Response compression

### ISSUE THREAD

**Issue by solvingproblemswithtechnology** [DAXGRID/typesense-dotnet#205]
Hi!

I just wanted to share how I enabled reponse compression for Typesense.

```
builder.Services.AddTypesenseClient(config =>
{
    builder.Configuration.Bind("Typesense", config);
})
.AddHttpClient<ITypesenseClient, TypesenseClient>(c =>
{
    c.DefaultRequestHeaders.AcceptEncoding.Add(new("br"));
    c.DefaultRequestHeaders.AcceptEncoding.Add(new("gzip"));
    c.DefaultRequestHeaders.AcceptEncoding.Add(new("deflate"));
})
.ConfigurePrimaryHttpMessageHandler(config => new HttpClientHandler
{
    AutomaticDecompression = DecompressionMethods.Brotli | DecompressionMethods.Deflate | DecompressionMethods.GZip
});
```

This way our response times got better, specially for big payloads. It would be nice to either have this in the docs or an option in AddTypesenseClient to enable it.

Keep up the good job!

**Comment by runeanielsen** [comment]
Hi @solvingproblemswithtechnology,

Feel free to make a PR with an example of how to do it in the README.

// Rune

**Comment by runeanielsen** [comment]
@solvingproblemswithtechnology this is now an option, implemented by @Henr1k80 in #214 , I'll be released in the next version.
